engine/src/main/java/org/terasology/logic/console/ConsoleImpl.java

```diff
@@ -117,6 +117,30 @@ private void addErrorMessage(String message) {
         addMessage(new Message(message, CoreMessageType.ERROR));
     }
 
+    /**
+     * Adds a message to the console
+     *
+     * @param message    The message to be added
+     * @param newLine    Flag indicates about new line at the of message
```

Comment:
Same as previous comments. I'd also write:

`@param message    The message to be added, as a string.`